docs/components/concepts/process-instance-migration.md

```diff
@@ -202,17 +202,13 @@ After migrating active element `A`, the process instance is no longer subscribed
 ![After migrating back, the process instance is no longer subscribed to the message boundary event](assets/process-instance-migration/migration-boundary-event_before.png)
 
 :::note
-At this time, you cannot provide mapping instructions for boundary events yet.
-Additionally, only message boundary events are supported in process instance migrations.
+At this time, only message boundary events are supported in process instance migrations.
 Other types of boundary events will be supported in future versions.
 :::
 
 :::tip
-Currently, you cannot migrate an active element with a message boundary event attached to an element that also has a message boundary event attached with the same message name.
-While we're working on resolving this, a workaround is available by splitting the migration into two steps:
-
-First migrate the process instance to a process model where the element is without a message boundary event.
-Once the process instance is successfully migrated, you can migrate the process instance to a process model where the message boundary event is attached to the element again.
+Currently, you cannot migrate an active element with a message boundary event attached to an element that also has a message boundary event attached if both the boundary events rely on the same message name and no mapping is provided between these boundary events.
+While we're working on resolving this, you can migrate this case by providing a mapping between the boundary events.
```

Comment:
Do you mean "While we're working on resolving this, you can **mitigate** this case by providing a mapping between the boundary events."?

Reply:
@christinaausley migrate here referred to the topic `process instance migration`. The intention is to describe a workaround so users can still migrate their process instance even with this limitation. Alternative wording could be:

```
While we're working on resolving this, you can provide a mapping between the boundary events to workaround this limitation.
```